### PR TITLE
Documenting usage of positional params in reopenClass

### PR DESCRIPTION
**Pull request by thriqon**
updates #482 due to https://github.com/emberjs/ember.js/commits/a7146f443ec611b31d2db8d5cd3bbc6fcc26b3a9
also see https://github.com/emberjs/ember.js/commit/995e2d2e0d5e6ae54afb6f3095c3d1efb20cdcfc#commitcomment-12513902

Should this be based on the 2.0.0 branch since this behavior is already present in v2?

### CODE REVIEW

File: source/components/passing-properties-to-a-component.md [hunk 1]
```diff
@@ -80,14 +80,18 @@ pass properties to the chosen component in the same manner:
...
 Apart from passing attributes to a component, you can also pass in positional parameters, like we've seen with the `{{link-to}}`, e.g. `{{link-to "user" userModel}}`.
 
-You can access these parameters by setting the `positionalParams` attribute in your component.
+You can access these parameters by setting the `positionalParams` attribute in your component class.
 
 ```app/components/x-visit.js
 import Ember from 'ember';
 
-export default Ember.Component.extend({
+let myComponent = Ember.Component.extend();
```
Comment (locks) [inline source/components/passing-properties-to-a-component.md]: `const MyComponent` instead

**Comment by thriqon** [comment]
@locks I just changed to `const`

**Comment by locks** [comment]
@thriqon you missed the capitalized MyComponent class

**Comment by thriqon** [comment]
sorry for that

**Comment by locks** [comment]
@thriqon :+1: 
@rwjblue can you review? this is unfamiliar to me

**Comment by rwjblue** [comment]
Content looks good, needs a rebase.

**Comment by thriqon** [comment]
@rwjblue I rebased the commit :)

**Comment by locks** [comment]
:boom: :hammer:

**Pull request merged**